backend/data/src/test/kotlin/io/tolgee/unit/formats/apple/in/XcstringsFormatProcessorTest.kt

```diff
@@ -0,0 +1,129 @@
+package io.tolgee.unit.formats.apple.`in`
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import io.tolgee.formats.apple.`in`.xcstrings.XcstringsFileProcessor
+import io.tolgee.testing.assert
+import io.tolgee.unit.formats.PlaceholderConversionTestHelper
+import io.tolgee.util.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class XcstringsFormatProcessorTest {
+    lateinit var mockUtil: FileProcessorContextMockUtil
+
+    @BeforeEach
+    fun setup() {
+        mockUtil = FileProcessorContextMockUtil()
+        mockUtil.mockIt("example.xcstrings", "src/test/resources/import/apple/example.xcstrings")
+    }
+
+    @Test
+    fun `returns correct parsed result`() {
+        processFile()
+        mockUtil.fileProcessorContext.assertLanguagesCount(2)  // en, fr
```

Comment:
I would like to see that by default it converts `%@` and `%lld` parameters into the Tolgee Universal ICU format by default. Please add such params to the example files and test them.